azure-pipelines.yml

```diff
@@ -44,54 +44,54 @@ jobs:
       - bash: echo "##vso[task.prependpath]/usr/share/miniconda/bin"
         displayName: "Add conda to PATH on Linux"
         condition: eq( variables['Agent.OS'], 'Linux' )
-      - script: |
-          CALL conda update -n base conda -y
-          CALL conda init --all
+      - bash: |
+          conda update -n base conda -y &&
+          conda init --all
         displayName: "Conda setup on Windows"
         condition: eq( variables['Agent.OS'], 'Windows_NT' )
       - bash: |
-          sudo conda update -n base conda -y
+          sudo conda update -n base conda -y &&
           sudo conda init bash
         displayName: "Conda setup on Linux"
         condition: eq( variables['Agent.OS'], 'Linux' )
       - bash: |
-          conda --version
-          conda env create --file environment.yml
-          source activate qcodes
-          pip install -r test_requirements.txt
-          pip install -r docs_requirements.txt
+          conda --version &&
+          conda env create --file environment.yml &&
+          source activate qcodes &&
+          pip install -r test_requirements.txt &&
+          pip install -r docs_requirements.txt &&
           pip install -e .
         displayName: "Install environment, qcodes"
       - bash: |
-          source activate qcodes
+          source activate qcodes &&
           mypy qcodes
         displayName: "mypy"
       - bash: |
-          source activate qcodes
-          cd ..
-          git clone https://github.com/QCoDeS/qcodes_generate_test_db.git
-          cd qcodes_generate_test_db
-          python generate_version_0.py
-          python generate_version_1.py
-          python generate_version_2.py
-          python generate_version_3.py
-          python generate_version_4a.py
-          python generate_version_4.py
-          python generate_version_5.py
-          python generate_version_6.py
+          source activate qcodes &&
+          cd .. &&
+          git clone https://github.com/QCoDeS/qcodes_generate_test_db.git &&
+          cd qcodes_generate_test_db &&
+          python generate_version_0.py &&
+          python generate_version_1.py &&
+          python generate_version_2.py &&
+          python generate_version_3.py &&
+          python generate_version_4a.py &&
+          python generate_version_4.py &&
+          python generate_version_5.py &&
+          python generate_version_6.py &&
           python generate_version_7.py
         displayName: "Generate db fixtures"
         condition: succeededOrFailed()
-      - script: |
-          CALL C:\Miniconda\condabin\conda_hook.bat
-          CALL conda activate qcodes
-          cd qcodes
+      - bash: |
+          C:\Miniconda\condabin\conda_hook.bat &&
```

Comment:
Running a bat script from bash cannot work

Reply:
For bash we are still using the old fashioned way of activating the env with source. 
Running the bash script that enables `conda init` in a platform independent way seems more complicated than it should be as the location is platform dependent and you need sudo on linux but sudo is not available in git bash on windows

Reply:
> Running a bat script from bash cannot work
yep, learned it the hard way. also that conda_hook.bat is not meant for bash... so, lack-of-knowledge everywhere :)

as for `conda init` - such a shame :( but ok. hope the `source activate` will be allowed for some time by conda devs.